### ISSUE TITLE
Honor metadata orientation information

### ISSUE THREAD

**Issue by di72nn** [VincentEngel/VES-Image-Compare#57]
Sometimes gallery apps and image viewers display a photo as expected, but VES Image Compare displays in turned 90/180 degrees.
I haven't looked into details, but I assume it has something to do with handling orientation information from EXIF (or whatever else type of metadata there is).

**Comment by VincentEngel** [comment]
Yes, the app does not check for orientation and just uses it as it is loaded. I looked into this once, but didn't find the correct solution for this problem. Maybe I will look into it at a later point of time, until then please use the rotation button.
